Mu2eKinKal/fcl/prolog.fcl

```diff
@@ -396,7 +395,7 @@ Mu2eKinKal : {
     MaterialSettings : @local::Mu2eKinKal.MAT
     KKFitSettings: {
       @table::Mu2eKinKal.KKFIT
-      SampleSurfaces : ["TT_Front","TT_Mid","TT_Back","ST_Outer","ST_Back","OPA","TSDA"]
+      SampleSurfaces : ["ST_Outer","ST_Back"] # these are additional surfaces; surfaces used in extrapolation are also sampled
```

Comment:
Are the three removed surfaces implicitly present or are they removed.  If implicit please update the comment to say so.

Reply:
The surfaces are explicit in the extrapolation configuration. These additional surfaces will all go away once the extrapolation codes are complete.